_topics/en/anonymity-networks.md

```diff
@@ -0,0 +1,139 @@
+
+  - title:  Bitcoin Core PR Review Club on a proposed implementation of BIP155 addr v2
+    url: /en/newsletters/2020/08/12/#bitcoin-core-pr-review-club
+
+  - title: BIP for new address relay message to support Tor v3 onion addresses
+    url: /en/newsletters/2019/03/12/#version-2-addr-message-proposed
+
+  - title: Bitcoin Core 0.21.0 released with support for Tor v3 onion addresses
+    url: /en/newsletters/2021/01/20/#bitcoin-core-0-21-0
+
+  - title: "Year-in-review: support for Tor v3 onion addresses"
+    url: /en/newsletters/2020/12/23/#addrv2
+
+  - title: "Bitcoin Core #19991 enables tracking inbound connections from onion peers"
+    url:  /en/newsletters/2020/10/07/#bitcoin-core-19991
+
+  - title: Desktop version of Blockstream Green released with Tor support
+    url: /en/newsletters/2020/06/17/#desktop-version-of-blockstream-green-wallet
+
+  - title: Presentation on enhancements in LND 0.10, including better Tor support
+    url: /en/newsletters/2020/05/06/#lnd-v0-10
+
+  - title: "BOLTs #751 updates BOLT7 to better handled multi-network node announcements"
+    url: /en/newsletters/2020/03/25/#bolts-751
+
+  - title: CKBunker allows specifying spending conditions for a Tor-enabled Coldcard
+    url: /en/newsletters/2020/02/19/#ckbunker-using-psbts-for-an-hsm
+
+  - title: "Eclair #1278 allows users to skip using SSL when using an onion service"
+    url: /en/newsletters/2020/02/05/#eclair-1278
+
+  - title: "C-Lightning #3155 adds option to use a static onion service address"
+    url: /en/newsletters/2019/12/11/#c-lightning-3155
+
+  - title: "Blockstream Green Tor built-in Tor support for both iOS and Android"
+    url: /en/newsletters/2019/10/23/#blockstream-green-tor-support
+
+  - title: "BIPs #766 assigns BIP155 to the addr v2 proposal for v3 onion addresses"
+    url: /en/newsletters/2019/07/31/#bips-766
+
+  - title: "Bitcoin Core #15651 always binds to the default port when listening on Tor"
+    url: /en/newsletters/2019/06/26/#bitcoin-core-15651
+
+  - title: "Eclair #736 adds support for using and becoming an onion service"
+    url: /en/newsletters/2019/02/12/#eclair-736
+
+  - title: "LND #1516 adds support for automatically setting up a v3 onion service"
+    url: /en/newsletters/2018/09/18/#lnd-1516
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: Dandelion
+    link: topic dandelion
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Anonymity networks** are systems that allow network communication
+  without senders or receivers needing to reveal their IP addresses to
+  each other.  The best known of these are Tor and I2P.
+
+---
+
+The use of anonymity networks can go a long way to improving the privacy
+of some Bitcoin software, but it can also be a liability in some cases.
+Some examples:
+
+- **Sending your own transactions** benefits immensely from using an
```

Comment:
I'm not sure what these bullet points are listing. Is it "liabilities" (in which case "sending your own transaction" shouldn't be included). I think they're all good points, but I don't see how they naturally fit into a single list.